lib/settings.js

```diff
@@ -0,0 +1,36 @@
+var fs = require('fs');
+var AppDirectory = require('appdirectory');
+var dirs = new AppDirectory('sql-seer');
```

Comment:
This could probably be merged into config.json, its the thing that loads up settings at startup and does get things from multiple files. I get where you're going with this though.